index.html

```diff
@@ -1,11 +1,125 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
-    <h1>Testing</h1>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header>
+        <h1 id="title">Survey Form</h1>
+    </header>
+    <main class="container">
+        <p id="description">Let us know how we can improve freeCodeCamp</p>
+        <form id="survey-form" action="index.html" method="post">
+            <fieldset>
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="name-label">Name: *</label>
+                    </div>
+                    <div class="right-row">
+                        <input class="input-field" type="text" id="name" placeholder="Enter your name" required>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="email-label">* Email:</label>
+                    </div>
+                    <div class="right-row">
+                        <input class="input-field" type="email" id="email" placeholder="Enter your Email" required>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="number-label">* Age:</label>
+                    </div>
```

Comment:
# Form label 

It will be nice to have the asterisk sign after the label text

Reply:
- [ ] You used good way of communication. please keep it up. I suggest it would be great if you give a bit details why you want asterisk be used here. giving details gives the student a path and can quickly realize the mistakes.

## For example:👇

1. This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

1. This request lacks a "why": Why don't you use an article tag here instead of a div?